r/R/ipc_stream.R

```diff
@@ -68,8 +68,8 @@ write_ipc_stream <- function(x, sink, ...) {
 #' @return A `raw` vector containing the bytes of the IPC serialized data.
 #' @examplesIf arrow_available()
 #' # The default format is "stream"
-#' write_to_raw(mtcars)
-#' write_to_raw(mtcars, format = "file")
+#' mtcars_raw <- write_to_raw(mtcars)
+#' mtcars_raw_file <- write_to_raw(mtcars, format = "file")
```

Comment:
TBH I don't think we need both of these (and this isn't really a part of our API we want to advertise, it's to support a quirk of something that sparklyr depended on)

```suggestion
```